src/main/java/htsjdk/samtools/reference/FastaSequenceFile.java

```diff
@@ -56,9 +60,21 @@ public FastaSequenceFile(final File file, final boolean truncateNamesAtWhitespac
     public FastaSequenceFile(final Path path, final boolean truncateNamesAtWhitespace) {
         super(path);
         this.truncateNamesAtWhitespace = truncateNamesAtWhitespace;
+        this.seekableStream = null;
```

Comment:
Maybe a way to clean up this code is to open the `java.nio.Path` as a `SeekableStream` here and initialize the field.

Reply:
This would probably work, but I was being cautious and leaving the old implementation unchanged.